crawl/dependency.py

```diff
@@ -123,6 +123,10 @@ def references_artifact(self):
     def bazel_buildable(self):
         raise Exception("must be implemented in subclass")
 
+    @property
+    def override_matching_str(self):
+        raise Exception("must be implemented in subclass")
+
```

Comment:
Is this the value of the rhs in the "overrides file"?  Maybe this can be called override_target in that case?  Please document this attribute, see large comment on top of the file.

Reply:
Also, please add something to dependencytest for this.